### PR TITLE
store: add some log.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @hhkbp2 @zhangjinpeng1987

### CODE REVIEW

**Comment by hhkbp2** [comment]
LGTM

**Comment by BusyJay** [comment]
LGTM

File: src/raftstore/store/peer_storage.rs [hunk 1]
```diff
@@ -459,9 +463,14 @@ impl PeerStorage {
                                  &raft_end,
                                  &mut |key, _| {
                                      try!(wb.delete_cf(*handle, key));
+                                     raft_count += 1;
                                      Ok(true)
                                  }));
-
+        info!("{} clean peer {} meta keys and {} raft keys, takes {:?}",
```
Comment (ngaut) [inline src/raftstore/store/peer_storage.rs]: s/clean peer/clear peer,/

**Comment by ngaut** [comment]
LGTM

**Pull request merged**